static/css/restyle.less

```diff
@@ -580,6 +580,15 @@ button.search-button {
   margin-top: 0;
 }
 
+.toplist li a {
+  line-height: 1.4em;
+  padding: .125em 0 .25em;
```

Comment:
I agree with the leading zeros, and I'm sure some optimiser strips them anyway.

The rest of the file didn't use them either so I went with this to be consistent, I think. But we can move to leading zeros everywhere, sure. (Filed #1944)